src/features/emojiMod.ts

```diff
@@ -65,32 +65,28 @@ const handleReport = (
 };
 
 const reactionHandlers: ReactionHandlers = {
-  "⚠️": (reaction, message, member) => {
+  "⚠️": async (reaction, message, member) => {
     // Skip if the post is from someone from the staff
-    if (
-      !message.guild ||
-      !message.author ||
-      isStaff(message.guild.member(message.author.id))
-    ) {
+    const { guild, author } = message;
+    if (!guild || !author || isStaff(await guild.members.fetch(author.id))) {
       return;
     }
     // If the user that reacted isn't in the staff, remove the reaction, send a
     if (!isStaff(member)) {
       reaction.users.remove(member.id);
-      member.send([
-        "Hey there! 👋",
-        "The ⚠️ reaction is reserved for staff usage as part of our moderation system.  If you would like to mark a message as needing moderator attention, you can use react with 👎 instead.",
-        "Thanks!",
-      ]);
+      member.send(`Hey there! 👋
+The ⚠️ reaction is reserved for staff usage as part of our moderation system.  If you would like to mark a message as needing moderator attention, you can use react with 👎 instead.
+Thanks!
+`);
```

Comment:
This changes it from 3 messages to 1 message with 3 lines, not sure if that was intentional.